### PR TITLE
Merge master into features

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: doc/en/skipping.rst [hunk 1]
```diff
@@ -179,16 +179,15 @@ information.
 Skipping on a missing import dependency
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can use the following helper at module level
-or within a test or test setup function:
+You can skip tests on a missing import by using :ref:`pytest.importorskip ref`
```
Comment (blueyed) [inline doc/en/skipping.rst]:
```suggestion
You can skip tests on a missing import by using :ref:`pytest.importorskip`
```
?

File: doc/en/skipping.rst [hunk 2]
```diff
@@ -179,16 +179,15 @@ information.
...
+at module level or within a test or test setup function.
```
Comment (blueyed) [inline doc/en/skipping.rst]:
```suggestion
at module level, within a test, or test setup function.
```

File: src/_pytest/outcomes.py [hunk 3]
```diff
@@ -161,10 +161,17 @@ def importorskip(
     if the module cannot be imported.
 
     :param str modname: the name of the module to import
-    :param str minversion: if given, the imported module ``__version__`` attribute must be
-        at least this minimal version, otherwise the test is still skipped.
-    :param str reason: if given, this reason is shown as the message when the module
-        cannot be imported.
+    :param str minversion: if given, the imported module ``__version__``
```
Comment (blueyed) [inline src/_pytest/outcomes.py]:
```suggestion
    :param str minversion: if given, the imported module's ``__version__``
```

File: src/_pytest/pytester.py [hunk 4]
```diff
@@ -630,6 +630,12 @@ def mkpydir(self, name):
         return p
 
     def copy_example(self, name=None):
+        """Copy file from project's directory into the testdir.
+
+        :param str name: The name of the file for copy.
```
Comment (blueyed) [inline src/_pytest/pytester.py]:
```suggestion
        :param str name: The name of the file to copy.
```

**Comment by nicoddemus** [comment]
Thanks!

File: doc/en/reference.rst [hunk 5]
```diff
@@ -27,6 +27,8 @@ pytest.skip
 
 .. autofunction:: _pytest.outcomes.skip(msg, [allow_module_level=False])
 
+.. _`pytest.importorskip ref`:
```
Comment (blueyed) [inline doc/en/reference.rst]:
```suggestion
.. _`pytest.importorskip`:
```

To fix:

> Warning, treated as error:
/home/travis/build/pytest-dev/pytest/doc/en/skipping.rst:182:undefined label: pytest.importorskip (if the link has no caption the label must precede a section header)

That's why I had a "?" with the suggestion.. ;)

**Comment by nicoddemus** [comment]
Hmmm nevermind this branch then, I will re-do the merge once we merge the pending PRs. 👍

**Pull request closed without merging**